todo/js/script.js

```diff
@@ -1,16 +1,49 @@
-const app = new function IIFE() {
+function Todo(config) {
+    // Accepting and preseting arguments in object format - can't deconstruct as argument in case of it was undefined
+    /*
+        config : {
+            id: String,
+            shouldInit: Boolean
+        }
+    */
+    if (config == undefined) {
+        config = {
+            id: 'app',
+            shouldInit: true
+        }
+    }
+    // if Only one was undefined
+    if (config.id == undefined) config.id = 'app';
+    if (config.shouldInit == undefined) config.shouldInit = true;
```

Comment:
I suggest to use [default value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) and [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) to handle default config.

Reply:
much appreciate the suggestion, what do you think of using `{id = 'app', shouldInit = true} = {}` as the first argument as I had in mind to accept an object for any sort of config?